README.md

```
@@ -212,8 +215,9 @@ comfy node [show|simple-show] [installed|enabled|not-installed|disabled|all|snap
 #### Unified Dependency Resolution (--uv-compile)
 
 Requires ComfyUI-Manager v4.1+. Instead of installing dependencies per-node with
-`pip install`, `--uv-compile` batch-resolves all custom node dependencies via
-`uv pip compile` to avoid version conflicts.
+`pip install`, `--uv-compile` delegates to ComfyUI-Manager's unified resolver which batch-resolves 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)